### PR TITLE
Rewrite README

### PR DESCRIPTION
**Pull request by tijmenb**
The readme currently doesn't explain how to set up Sidekiq and is incorrect about the required configuration.

This updates the readme, following a set up process with content-tagger.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,58 @@
 Provides a unified set of configurations and behaviours when using Sidekiq
```
Comment (gpeng) [inline README.md]: It would be useful to expand a bit on what the 'behaviours' are that it adds

Comment (tijmenb) [inline README.md]: Added some things.

**Comment by gpeng** [comment]
:+1:

**Comment by mobaig** [comment]
👍

**Pull request merged**